src/config.rs

```diff
@@ -37,6 +37,7 @@ pub struct Config {
     pub base_address_shift: u64,
     pub html_url: Option<url::Url>,
     /// Path to YAML file with chip-specific settings
+    #[cfg(feature = "yaml")]
     pub settings: Option<PathBuf>,
```

Comment:
I think this would just ignore the settings file, even if it is configured, correct?
Wouldn't it be better to report an error instead, like I implemented in #877?